### PR TITLE
More cleanups

### PR DESCRIPTION
**Pull request by set-killer**
As pointed out in #8.

### CODE REVIEW

File: godot_export_manager.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ class godot_export_manager(bpy.types.Panel):
     bpy.types.Scene.godot_export_on_save = BoolProperty(default=False)
 
     def draw(self, context):
-        """ Draw function for all ui elements """
+        # Draw function for all ui elements
```
Comment (akien-mga) [inline godot_export_manager.py]: Why? Those are docstrings and should stay in triple quotes.

Comment (set-killer) [inline godot_export_manager.py]: ok, will revert this changes

Comment (set-killer) [inline godot_export_manager.py]: done

Comment (set-killer) [inline godot_export_manager.py]: Thanks @akien-mga
